images/utils.go

```diff
@@ -21,6 +21,8 @@ type forEachImageFunc func(index int, image Image) error
 
 // getImage returns the image for the alert or an error. It returns a nil
 // image if the alert does not have an image token or the image does not exist.
+//
+//nolint:revive
```

Comment:
what rule is being silenced here? Can we add a nolint for a specific violation?

Reply:
It doesn't like the `if else` block here:

```go
if errors.Is(err, ErrImageNotFound) || errors.Is(err, ErrImagesUnavailable) {
	return nil, nil
} else if err != nil {
	l.Warn("failed to get image with token", "token", token, "error", err)
	return nil, err
} else {
	return img, nil
}
```

and wants me to rewrite it as:

```go
if errors.Is(err, ErrImageNotFound) || errors.Is(err, ErrImagesUnavailable) {
	return nil, nil
}
if err != nil {
	l.Warn("failed to get image with token", "token", token, "error", err)
	return nil, err
}
return img, nil
```